packages/core/application/core-application-browser-internal/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { ApplicationService } from './src/application_service';
+export { CoreScopedHistory } from './src/scoped_history';
+export type {
+  InternalApplicationSetup,
```

Comment:
Given the updated package generation script is now configured to have `{package}/index.ts` as the entrypoint, I changed my approach to have it import everything directly, without having a `{package}/src/index.ts` too, as the 2 files felt like duplication (the higher-level one was re-exporting everything exported from the second one)

Reply:
Are we sure we want to take this approach? IMO, having a root `index.js` file for each package subfolder is a better dev experience, since it's easier to find which files contain different implementations within the subfolder.

Reply:
> Are we sure we want to take this approach?

No, not at all. TBH I'm mostly testing things. For the server-side SO domain, I did have a the root index file that replicated the export from the `src/index` file, but it felt like unnecessary duplication. OTOH I agree that having an index file per sub folder is the common approach we've been doing, so 🤷